package.json

```diff
@@ -18,7 +18,7 @@
     "start": "nest start",
     "__predev": "npm run generate-database-types",
     "prestart:debug": "npm run generate-database-types",
-    "dev": "dotenv -e .env/.env.local nest start --watch",
```

Comment:
this didn't fix your debug issue, so lets remove this change so other devs do not get confused by the server not restarting anymore

Reply:
This --watch flag did never work